cmd/tests/cmd_cloud_run_test.go

```diff
@@ -10,3 +15,44 @@ func TestK6CloudRun(t *testing.T) {
 func setupK6CloudRunCmd(cliFlags []string) []string {
 	return append([]string{"k6", "cloud", "run"}, append(cliFlags, "test.js")...)
 }
+
+func TestCloudRunCommandIncompatibleFlags(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		name               string
+		cliArgs            []string
+		wantStderrContains string
+	}{
+		{
+			name:               "using --linger should be incompatible with k6 cloud run",
+			cliArgs:            []string{"--linger"},
+			wantStderrContains: "the --linger flag can only be used in conjunction with the --local-execution flag",
+		},
+		{
+			name:               "using --exit-on-running should be incompatible with k6 cloud run --local-execution",
+			cliArgs:            []string{"--local-execution", "--exit-on-running"},
+			wantStderrContains: "the --local-execution flag is not compatible with the --exit-on-running flag",
+		},
+		{
+			name:               "using --show-logs should be incompatible with k6 cloud run --local-execution",
+			cliArgs:            []string{"--local-execution", "--show-logs"},
+			wantStderrContains: "the --local-execution flag is not compatible with the --show-logs flag",
+		},
+	}
+
+	for _, tc := range testCases {
+		tc := tc
+
+		t.Run(tc.name, func(t *testing.T) {
+			t.Parallel()
+
+			ts := getSimpleCloudTestState(t, nil, setupK6CloudRunCmd, tc.cliArgs, nil, nil)
+			ts.ExpectedExitCode = -1
```

Comment:
```suggestion
			ts.ExpectedExitCode = -1
```
Shouldn't be an InvalidConfig type instead of generic `-1`?

Reply:
Double on that, it's probably just switching to `errext.WithExitCodeIfNone` for errors

Reply:
Good catch, done 👍🏻